rpc/transaction_test.go

```diff
@@ -304,11 +265,11 @@ func TestTransactionReceipt(t *testing.T) {
 		"mainnet": {},
 		"integration": {
 			{
-				TxnHash: utils.TestHexToFelt(t, "0x49728601e0bb2f48ce506b0cbd9c0e2a9e50d95858aa41463f46386dca489fd"),
+				TxnHash: utils.TestHexToFelt(t, "0xf2f3d50192637e8d5e817363460c39d3a668fe12f117ecedb9749466d8352b"),
```

Comment:
integration is different network than Sepolia, so lets just remove this altogether. Sorry about that!